modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-type-grid/src/main/resources/META-INF/resources/grid_field.js

```diff
@@ -121,24 +100,20 @@ AUI.add(
 						container.all('.help-block').appendTo(container.one('.form-group'));
 					},
 
-					_getGridRowsNode: function() {
+					_getFocusTarget: function() {
 						var instance = this;
 
-						var container = instance.get('container');
-
-						var gridRowsNode = container.all('tbody tr');
-
-						return gridRowsNode;
+						return instance.focusTarget;
```

Comment:
I just tested and the focus was not working. See: https://drive.google.com/file/d/0B-Sc8uDOPJSucmxGX1cwdmZSYVE/view?usp=sharing

I sen you a PR here: https://github.com/diogo-lins/liferay-portal/pull/44, it looks like it works now: https://drive.google.com/file/d/0B-Sc8uDOPJSuajd4QTM0ZlYzaTA/view?usp=sharing

This should fix the focus. Please review my changes. I think we can clean it up a little more now because we can remove the autofocus from the soy.
 
Thx